gitwatch.sh

```diff
@@ -56,6 +56,7 @@ shelp () { # Print a message about how to use this script
     echo " -d <fmt>         the format string used for the timestamp in the commit"
     echo "                  message; see 'man date' for details; default is "
     echo "                  \"+%Y-%m-%d %H:%M:%S\""
+    echo " -p <remote>"
```

Comment:
`-p` is just a duplicate of `-r`. I believe we should leave it undocumented and deprecate `-p` to avoid confusion.